### ISSUE TITLE
sagan-site build error

### ISSUE THREAD

**Issue by hanmomhanda** [spring-io/sagan#624]
One test fails while local build as follows:

![20160210_sagan-site-test-error-01](https://cloud.githubusercontent.com/assets/355249/12921359/49c7c71c-cf90-11e5-9673-347756d2ccbf.png)

![20160210_sagan-site-test-error-02](https://cloud.githubusercontent.com/assets/355249/12921365/52e783be-cf90-11e5-9daf-b96cb3282765.png)

I guess the reason it fails as follows:

`sagan.blog.support.PostView` is **already compiled with the default locale**, so it does not produce "July 02, 2012" but "07월 02, 2012", when it is invoked by the test method, `sagan.blog.support.PostViewTests.formattedPublishDateForPublishedPosts()` with **the locale being set to Locale.US temporarily**.

**Comment by pintiliea** [comment]
It's this line in [PostViewTests.java](https://github.com/spring-io/sagan/blob/master/sagan-site/src/test/java/sagan/blog/support/PostViewTests.java)

```java
assertThat(postView.getFormattedPublishDate(), equalTo("July 02, 2012"));
```
implemented like this in [PostView.java](https://github.com/spring-io/sagan/blob/master/sagan-site/src/main/java/sagan/blog/support/PostView.java)

```java
return post.isScheduled() ? "Unscheduled" : DATE_FORMAT.format(post.getPublishAt());
```
where 
```java
SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("MMMM dd, yyyy");
```

It looks like it was fixed by #214  *Fix tests when default Locale is not "US" (a: bug)* and maybe reintroduced by #374 *Continue migration to JDK 8 Date and Time API (a: enhancement)*

If you want to test the formatting of dates, the test should be initialized with Locale.US.

**Comment by bclozel** [comment]
This has been fixed with #688.
